packages/api-explorer/src/components/SchemaTabs/index.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { FormattedMessage } from 'react-intl';
+
+import BlockWithTab from '../BlockWithTab'
+
+const ResponseSchema = require('../../ResponseSchema');
+const RequestSchema = require('../../RequestSchema');
+
+const styleList = {
+  fontSize: '18px',
+  color: '#aeaeae',
+  textTransform: 'uppercase',
+  borderBottom: `1px solid #ddd`,
+  fontWeight: 'bold',
+  background: 'none',
+  padding: 0
+}
+const styleSelectedItem = {
+  color: '#aeaeae',
+  background: 'none',
+  borderBottom: '3px solid #aeaeae'
+}
+const styleLink = {
+  color: 'inherit'
+}
+const styleItem = {
+  borderBottom: '3px solid rgba(255,255,255,0)',
+  color: '#c4c4c4'
+}
+
+function renderMissingSchema(nameSchema) {
+  return (
+    <div style={{
+      padding: 10,
+      background: '#ededed'
+    }}
+    >
+      <FormattedMessage id={`schemaTabs.missing.${nameSchema}`} />
+    </div>
+  )
+}
+export default class SchemaTabs extends Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      selected: 'request'
+    }
+  }
+
+  renderResponseSchema() {
+    const { operation, oas } = this.props
+    return (
+      operation &&
+        operation.responses ? (
+          <ResponseSchema operation={operation} oas={oas} />
+        )
+        : renderMissingSchema('response')
+    )
+  }
+
+  renderRequestSchema() {
+    const { operation, oas } = this.props
+    return (
+      operation &&
+        operation.requestBody ? (
+          <RequestSchema operation={operation} oas={oas} />
+        )
+        : renderMissingSchema('request')
+    )
+  }
+
+  render() {
+    const { selected } = this.state
+    return (
+      <BlockWithTab
+        items={[{ value: 'request', label: 'request' }, { value: 'response', label: 'response' }]}
```

Comment:
Why label is in this mode? They should not be capitalized?

Reply:
Capitalization should be done in the style I'd not change the labels here but inside the block with tab so that every BlockWithTab has consistent headings.

Reply:
No, they are all upperCase, the same style of Description and Body Parameters.

Reply:
As far as capitalization is managed with styles and not data I'm fine with it! ;)